### ISSUE TITLE
CAS notarize fails with hash/name

### ISSUE THREAD

**Issue by pvizeli** [codenotary/cas#237]
**What happened**

![image](https://user-images.githubusercontent.com/15338540/139910275-f6522776-40e4-4c53-9551-2477cdc08adc.png)

**What you expected to happen**

That it works

**How to reproduce it (as minimally and precisely as possible)**

`cas n --hash xxx --name test`

**Environment**
```shell
# run "cas info" and copy/paste the output here
Error: unknown command "info" for "cas"
Run 'cas --help' for usage.
```

**Additional info (any other context about the problem)**

I have written a migration script to get the data from old backend and upload into CAS.
https://pastebin.com/qgQ2FBT1

**Comment by qrdl** [comment]
Problem reproduced, investigating

**Comment by qrdl** [comment]
Fixed in PR #238
